### DISCUSSION TITLE
Team A - Course 1

### DISCUSSION THREAD

**Discussion by VSFe** [lablup/contribution-academy-2021#40] [Show and tell]
## Course 1: Backend.ai 사용해보기
- **executed time (for training only)**

![course1_1](https://user-images.githubusercontent.com/4595546/129047973-e9df8cb5-669b-4fc8-9434-db401e2d5b91.PNG)

- **accuracy**

![course1_2](https://user-images.githubusercontent.com/4595546/129048016-1e90cf1d-8ac4-4096-98ee-d1523eb6e356.PNG)

- **total allocated resources**

![course1_3](https://user-images.githubusercontent.com/4595546/129048043-1db865a0-da9b-4b8a-9c9f-42229bbefd98.PNG)

**Comment by eunjungchoi** [comment]
최은정  (Team A)

### **Course 1: Backend.ai 사용해보기**

**executed time (for training only)**

<img width="809" alt="스크린샷 2021-08-11 오후 10 54 59" src="https://user-images.githubusercontent.com/5889435/129042475-df3bb808-0e24-4678-9835-940933922318.png">

**accuracy**

<img width="488" alt="스크린샷 2021-08-11 오후 10 55 26" src="https://user-images.githubusercontent.com/5889435/129042524-baf936c4-caec-4d33-844a-c8724b2bf224.png">

**total allocated resources:**

<img width="1107" alt="스크린샷 2021-08-11 오후 10 56 31" src="https://user-images.githubusercontent.com/5889435/129042571-98080dae-6f9b-49be-9314-ed9941533504.png">
